build.sbt

```diff
@@ -19,10 +19,11 @@ import scala.collection.immutable.ListSet
 
 import sbtcc._
 
-lazy val genManaged = taskKey[Seq[File]]("Generate managed sources and resources")
+lazy val genManaged = taskKey[Unit]("Generate managed sources and resources")
 lazy val genProps = taskKey[Seq[File]]("Generate properties scala source")
 lazy val genSchemas = taskKey[Seq[File]]("Generate DFDL schemas")
 lazy val genCExamples = taskKey[Seq[File]]("Generate C example files")
+lazy val genVersion = taskKey[Seq[File]]("Generate a VERSION file")
```

Comment:
Picky, I know, but the sentence should not use an indefinite article ('a').  We use the indefinite article when not knowing which one of several VERSION files we are referring to or showing that the file is one of a group of VERSION files.  There is only 'the' VERSION file, not 'a' VERSION file.  However, none of the other sentences use definite or indefinite articles, so let's drop the 'a'.